sympy/simplify/tests/test_simplify.py

```diff
@@ -1797,11 +542,6 @@ def _eval_simplify(x, **kwargs):
 def test_issue_from_PR1599():
     n1, n2, n3, n4 = symbols('n1 n2 n3 n4', negative=True)
     assert simplify(I*sqrt(n1)) == -sqrt(-n1)
-    assert (powsimp(sqrt(n1)*sqrt(n2)*sqrt(n3)) ==
```

Comment:
Just wanted to make sure for your focus at important points. I have split the test itself into two files one part remains here itself and other part moved to the `powsimp.py` same has been done for only one more `test-function` which involved `simplify